### PR TITLE
completed api-authorization

### PR DESCRIPTION
**Pull request by AndyPark20**
image

### CODE REVIEW

File: api-authorization/server/authorization-middleware.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const jwt = require('jsonwebtoken'); // eslint-disable-line
+const ClientError = require('./client-error'); // eslint-disable-line
+
+function authorizationMiddleware(req, res, next) {
+  /* your code here */
+  if(!req.headers){
```
Comment (Zhiwutian) [inline api-authorization/server/authorization-middleware.js:6]: you don't want to do a blanket check for headers, you want to try to access the token first and then check if there is a token good sir! If there is no token you want to send the Error

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny update!

File: api-authorization/server/authorization-middleware.js (hunk 1 above)
Comment (AndyPark20) [reply api-authorization/server/authorization-middleware.js:6]: Ah well noted Brett!

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**